### PR TITLE
Add missing changelog entries

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,10 +39,18 @@ Changed
 - Changed eauth "not enabled" log message level from debug to warning. (#50946)
 -  (#52546)
 - Refactored x509.certificate_managed to be easier to use. (#52935)
+- Don't log error when running "alternatives --display" on nonexistant target (#53911)
 - Improved logging for user auth issues. (#53990)
 - No longer emit extra logs when checking `alternatives.display` and `.check_exists`. (#53991)
 - Use lazy loading to get SLS data from master - significantly improves `state.apply` times when using gitfs with many branches. (#54468)
+- Blacken salt (Implements SEP 15). (#55765)
```
Comment (waynew) [inline CHANGELOG.md:46]: Not sure that this is /really/ a user-facing change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -38,10 +39,18 @@ Changed
...
+- Disable codecov PR comments and status checks (#55888)
```
Comment (waynew) [inline CHANGELOG.md:47]: Definitely not a user-facing change.

File: CHANGELOG.md [hunk 3]
```diff
@@ -38,10 +39,18 @@ Changed
...
 - Changed Salt icon for Windows. (#56194)
+- Replace `log.warn` by `log.warning` and deprecated `ElementTree.getchildren()` (#56226)
+- Docs Replace incorrectly used "it's" by "its" (#56249)
+- Docs: fix typo in windows-package-manager.rst (#56303)
+- Update `libnacl` to 1.7.1 (#56350)
+- Fix docs in win_wua (#56374)
```
Comment (waynew) [inline CHANGELOG.md:53]: I'm not sure how user-facing these changes are. Probably the libnacl one is good to keep.

File: CHANGELOG.md [hunk 4]
```diff
@@ -117,6 +126,7 @@ Fixed
 - Use `__utils__` for all registry calls. (#52992)
 - Added syndic log rotation to RPM. (#53040)
 - Use correct output in `zpool.present` when `test=true`. (#53145)
+- Fix dns-search is not created during ubuntu vm creation. (#55170, fixed #37709)
```
Comment (waynew) [inline CHANGELOG.md:129]: Should just be the # that was fixed, usually.
```suggestion
- Fix dns-search is not created during ubuntu vm creation. (#37709)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -126,7 +136,7 @@ Fixed
 - Fixed `guesseed` -> `guessed` typo in `archive` state. (#53480)
 - Fixed error with incorrect import statement masking real import error. (#53508)
 - Added some error handling around missing results from external returners. (#53517)
-- Changed to match repo paramter against repo name on `salt-run git_pillar.update`, so remote name can be used instead of full remote URL. (#53622)
+- Changed to match repo paramter against repo name on `salt-run git_pillar.update`, so remote name can be used instead of full remote URL. (#56605 - Master port of#53622)
```
Comment (waynew) [inline CHANGELOG.md:139]: No need to spell out the master port.

When someone goes to check that issue #, they'll see that there is a master port.

File: CHANGELOG.md [hunk 6]
```diff
@@ -213,22 +224,38 @@ Fixed
...
 - Fixed `panos` commit example in docs. (#56581)
 - Fixed issue with `salt.utils.functools.call_functions` not checking for expected arguments. (#56584)
 - Fixed a broken statement when using arbitrary `kwargs` in mine.value. (#56593)
+- Correct formatting issue with highstate output module (#56598)
 - Fixed support for booting VMs with UEFI on virt. (#56613)
+- Check kwargs for expected arguments in salt.utils.functools.call_function (#56635)
 - Updated old redirects and http->https fixes in docs. (#56655)
 - Renamed `salt/utils/docker/` to `salt/utils/dockermod/` to avoid clashes with the `docker` package from pypi. (#56669)
 - Changed behavior to implicitly ignore package epochs and just use the latest one. (#56681)
+- Fix btrfs test on Windows (#56688 - Master port of #52147)
```
Comment (waynew) [inline CHANGELOG.md:239]: Same here - no need to call out master port. Just put the original issue # if it exists, if there's no issue, then the original PR.

If there are multiple issues fixed in one PR, then I put the PR #.

File: CHANGELOG.md [hunk 7]
```diff
@@ -213,22 +224,38 @@ Fixed
...
 - Avoid throwing exception for missing security group in boto under test mode. (#56695)
+- Fix error retry handling for Route53 (#56696 - Master port of #51572)
```
Comment (waynew) [inline CHANGELOG.md:241]: same here - remove master port

File: CHANGELOG.md [hunk 8]
```diff
@@ -213,22 +224,38 @@ Fixed
...
 - Fix some function prompts in myssql module. (#56719)
+- Correct firewalld.reload_rules docstring (#56725)
 - Add appropriate comment for `svn export` state. (#56757)
+- Updated default master config file and updated the docs for issue #56053 (#56762)
+- Fix typo in vault docs (#56776 - Master port of #53964)
```
Comment (waynew) [inline CHANGELOG.md:246]: same here - remove master port

File: CHANGELOG.md [hunk 9]
```diff
@@ -213,22 +224,38 @@ Fixed
...
+- Silence some high-noise log messages in pillar cache (#56789 - Master port of #52763)
```
Comment (waynew) [inline CHANGELOG.md:247]: same here - remove master port

Comment (waynew) [reply CHANGELOG.md:247]: also should probably be in the "changed" section

File: CHANGELOG.md [hunk 10]
```diff
@@ -213,22 +224,38 @@ Fixed
...
+- Filter gluster CLI banner (fixes #52318) (#56794 - Master port of #52324)
```
Comment (waynew) [inline CHANGELOG.md:248]: This should just be `(#52318)`

File: CHANGELOG.md [hunk 11]
```diff
@@ -213,22 +224,38 @@ Fixed
...
+- Cleanup vault test (#56809)
```
Comment (waynew) [inline CHANGELOG.md:249]: Not a user-facing change.

File: CHANGELOG.md [hunk 12]
```diff
@@ -213,22 +224,38 @@ Fixed
...
 - Workaround upstream bug in jinja2 indent filter. (#56833)
+- Fixed issue when raid.destroy is called but zero-superblock is not executed (#56838 - Master port of #52138, #52137, #52133)
```
Comment (waynew) [inline CHANGELOG.md:251]: Same here - since this is multiple PRs, just call out the high-level #

File: CHANGELOG.md [hunk 13]
```diff
@@ -213,22 +224,38 @@ Fixed
...
 - Allow correct failure information to show up when calling `win_interfaces` (#56844)
+- Add a note about service.running (#56846 - Master port of #51511)
```
Comment (waynew) [inline CHANGELOG.md:253]: no need for "master port"

File: CHANGELOG.md [hunk 14]
```diff
@@ -213,22 +224,38 @@ Fixed
...
+- LocalClient: Pass input args/kwargs through salt.utils.args.parse_input; don't parse input twice and test fix. (#56853 - Master port of #49886 and #52709)
```
Comment (waynew) [inline CHANGELOG.md:254]: also remove "master port" - just link the one issue.

File: CHANGELOG.md [hunk 15]
```diff
@@ -237,18 +264,27 @@ Fixed
 - Provides better stacktrace in `win_pkg` return. (#56955)
 - Fixed `reg.present` to respect `(Default)` REG_SZ value of an empty string. (#56959)
 - OpenStack driver can now attach to multiple networks, also now respects provided `conn`. (#56960)
+- Fix condition for ConfigTestCase.test_datetime_config_validation (#56968)
```
Comment (waynew) [inline CHANGELOG.md:267]: if we're just updating a test case, and not changing code, no changelog entry is necessary.

File: CHANGELOG.md [hunk 16]
```diff
@@ -237,18 +264,27 @@ Fixed
...
 - Fixed literal comparsion in `user` state. (#56972)
+- Additional fixes for using cron state with non-root Minion (#56973)
+- Master port of the missing commit from PR #50109 (#57010)
```
Comment (waynew) [inline CHANGELOG.md:270]: I'd double-check this. If the PR is just fixing in-between release issues, it doesn't belong in the changelog. If this issue wasn't present in 3000.3 or earlier, then it's irrelevant to end users.

File: CHANGELOG.md [hunk 17]
```diff
@@ -237,18 +264,27 @@ Fixed
...
 - Fixed LazyLoader crashing when using ssh client via salt-api. (#57119)
+- Publisher ACL doc fixes (#57128 - Master port of #57128)
 - Fixed `acl.present` to properly detect changes for default ACLs and recursive folders. (#57147)
+- Fix pycrypto.gen_hash and update the test. (#57149)
+- Fixes to the docs for the Cloud VMWare topic. (#57177 - Master port of #49617)
 - Fixed Minion/Minon typo in docs. (#57181)
-- Fix UnicodeDecodeError when apply file.managed with binary contents in
-  test mode. (#57184)
+- Fix UnicodeDecodeError when apply file.managed with binary contents in test mode. (#57184)
+- Refactor SyncWrapper (#57188)
```
Comment (waynew) [inline CHANGELOG.md:284]: Might belong in `changes`, at least if the refactor should impact the user.

File: CHANGELOG.md [hunk 18]
```diff
@@ -304,21 +340,27 @@ Added
 - Added ability to pass arbitrary kwargs to zypper pkg. (#53693)
 - Added options for multi-use tokens for vault. (#54094)
 - Added devinfo module to get hardware information. (#54267)
+- Support list for `include_pat`/`exclude_pat` in `file.recurse`. (#54552 - Master port of #49837)
```
Comment (waynew) [inline CHANGELOG.md:343]: remove master port call-out

File: CHANGELOG.md [hunk 19]
```diff
@@ -304,21 +340,27 @@ Added
...
 - Adds versionlock plugin detection for yum/dnf. (#54798)
+- Add method to remove circular references in data objects and add test (#54930)
 - Improved nxos support. (#54931)
 - Added root and no_recommends parameters for Zypper and RPM. (#54954)
 - Added `token` parameter in `blkid`. (#54964)
+- Suppress pchanges output by adding new silent recurse option to file.directory state (#54976 - Master port of #49201)
```
Comment (waynew) [inline CHANGELOG.md:349]: Remove master-port call out

File: CHANGELOG.md [hunk 20]
```diff
@@ -304,21 +340,27 @@ Added
...
 - Added `cron.get_entry`. (#54985)
 - Added support for newer monit versions. (#55140)
 - Added btrfs and xfs as valid fstypes for parted and mkfs. (#55209)
 - Added functionality for `cmd.run_all` to accept a list when using powershell. (#55213)
 - Added Azure Blob Storage as an optional external pillar. (#55493)
 - Added ability to turn off FQDNs grains with `enable_fqdns_grains: False`. (#55581)
-- Added virt.*defined states. (#55814)
+- Added `virt.*defined` states. (#55814)
 - Add towncrier tool to the Salt project to help manage CHANGELOG.md file. (#55836)
+- Add selinux support to file.managed (#49860 - Master port of #49860)
```
Comment (waynew) [inline CHANGELOG.md:358]: remove master-port callout

File: CHANGELOG.md [hunk 21]
```diff
@@ -304,21 +340,27 @@ Added
...
 - Added hold and unhold support for `mac_brew_pkg`. (#55978)
+- Ensure the disk is free to use after seed.apply (#56004)
 - States/modules added for managing Helm. (#56081)
 - Added parallel run support for saltcheck. (#56097)
+- Documentation on `skip_grains` (#56098)
```
Comment (waynew) [inline CHANGELOG.md:363]: probably not necessary

File: CHANGELOG.md [hunk 22]
```diff
@@ -335,14 +377,24 @@ Added
 - Added support for rendering toml states. (#56615)
 - Added `set_path` option for salt-ssh shim. (#56627)
 - Added `win_wua.installed` to check a list of updates that apply to the current Windows build. (#56640)
+- Add gitfs and git_pillar fallback branch (#56647)
 - Added ability to compare package versions in Jinja templates. (#56678)
+- Feature to enable/disable termination protection for ec2 instances (#56703 - Master port of #51096)
+- Add function 'shutdown_host' to vmware cloud (#56711 - Master port of #50177)
+- Add a new drbd monitor function via drbdadm to deprecate the old method of drbd-overview (#56768 - Master port of #50410)
+- Add usergroup support to user.present (#56782 - Master port of #48704)
+- Add ARPCHECK option to rh7_eth template (#56801 - Master port of #49094)
```
Comment (waynew) [inline CHANGELOG.md:386]: I'm pretty sure this one is already in here?

Also several `master-port` callouts here

**Review by waynew — CHANGES_REQUESTED** [review]
Several changes here.

**Comment by s0undt3ch** [comment]
Updated

File: CHANGELOG.md [hunk 23]
```diff
@@ -117,6 +122,7 @@ Fixed
 - Use `__utils__` for all registry calls. (#52992)
 - Added syndic log rotation to RPM. (#53040)
 - Use correct output in `zpool.present` when `test=true`. (#53145)
+- Fix dns-search is not created during ubuntu vm creation. (#55170, fixed #37709)
```
Comment (waynew) [inline CHANGELOG.md:125]:
```suggestion
- Fix dns-search is not created during ubuntu vm creation. (#37709)
```

File: CHANGELOG.md [hunk 24]
```diff
@@ -214,22 +221,36 @@ Fixed
...
 - Fixed `panos` commit example in docs. (#56581)
 - Fixed issue with `salt.utils.functools.call_functions` not checking for expected arguments. (#56584)
 - Fixed a broken statement when using arbitrary `kwargs` in mine.value. (#56593)
+- Correct formatting issue with highstate output module (#56598)
 - Fixed support for booting VMs with UEFI on virt. (#56613)
+- Check kwargs for expected arguments in salt.utils.functools.call_function (#56635)
 - Updated old redirects and http->https fixes in docs. (#56655)
 - Renamed `salt/utils/docker/` to `salt/utils/dockermod/` to avoid clashes with the `docker` package from pypi. (#56669)
 - Changed behavior to implicitly ignore package epochs and just use the latest one. (#56681)
+- Fix btrfs test on Windows (#56688)
```
Comment (waynew) [inline CHANGELOG.md:236]: Just a test fix, not user-facing.

File: CHANGELOG.md [hunk 25]
```diff
@@ -214,22 +221,36 @@ Fixed
...
 - Avoid throwing exception for missing security group in boto under test mode. (#56695)
+- Fix error retry handling for Route53 (#56696)
 - Fix some function prompts in myssql module. (#56719)
+- Correct firewalld.reload_rules docstring (#56725)
 - Add appropriate comment for `svn export` state. (#56757)
+- Updated default master config file and updated the docs for issue #56053 (#56762)
+- Fix typo in vault docs (#56776)
+- Filter gluster CLI banner (fixes #52318) (#56794)
```
Comment (waynew) [inline CHANGELOG.md:244]: Already in here as #52318

**Review by waynew — CHANGES_REQUESTED** [review]
Looks like there may have been some confusion due to the port forward effort - I've noted a couple of things here, but we probably need to double check each of these to make sure they're not already in the changelog under the original issue #

File: CHANGELOG.md [hunk 26]
```diff
@@ -341,13 +351,18 @@ Added
 - Added `set_path` option for salt-ssh shim. (#56627)
 - Added `win_wua.installed` to check a list of updates that apply to the current Windows build. (#56640)
 - Added ability to compare package versions in Jinja templates. (#56678)
+- Add function 'shutdown_host' to vmware cloud (#56711)
+- Add a new drbd monitor function via drbdadm to deprecate the old method of drbd-overview (#56768)
```
Comment (waynew) [inline CHANGELOG.md:355]: Dupe of 50410

File: CHANGELOG.md [hunk 27]
```diff
@@ -341,13 +351,18 @@ Added
...
+- New requisite form: `onfail_all` (#47015)
```
Comment (waynew) [inline CHANGELOG.md:356]: Listed under 50264, since it also made other changes.

File: CHANGELOG.md [hunk 28]
```diff
@@ -341,13 +351,18 @@ Added
...
 - Add `auto_detect` feature for `ssh_ext_alternatives`. (#56894)
 - Add ability to display sys.doc style outputs but without actually loading the module. (#56902)
 - Added plist serializer. (#56954)
 - Added support for onedir/pop-build Salt in the `pip` module. (#56988)
 - Add support for disks volumes in virt.running state (#57005)
 - Add virt.all_capabilities helper function (#57009)
 - supervisord.status_bool method (#57049)
+- Uninstall msi-installed salt (#57068)
```
Comment (waynew) [inline CHANGELOG.md:364]: Listed as 56883 - this is probably the correct number, but that has the better description

File: CHANGELOG.md [hunk 29]
```diff
@@ -341,13 +351,18 @@ Added
...
+- RFC: Virtual module interface concept (#57093)
```
Comment (waynew) [inline CHANGELOG.md:365]: Non-code change (and should already just be in the SEP repo)

**Review by waynew — CHANGES_REQUESTED** [review]
One more batch :upside_down_face:

**Review by waynew — APPROVED** [review]

**Review by dwoz — APPROVED** [review]

**Pull request merged**